### DISCUSSION TITLE
How to get a custom header in the resolve method of rebing/graphql-laravel package in graphql?

### DISCUSSION THREAD

**Discussion by hardikpanotmc** [rebing/graphql-laravel#928] [General]
Hello everyone,

I am creating graphql in laravel. I was use rebing/graphql-laravel package( #link here ). I want to get a custom header in the resolve method of query/mutation. 

I want my header in the resolve method anyone has an idea about this then help me.

**Comment by mfn** [comment]
Have you tried calling the `request()` function to access the Request object?

**Reply by hardikpanotmc** [reply]
Oh, That's my bad luck. Thank you @mfn.
It's worked for me.
